sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -203,7 +203,13 @@ pi_result piDeviceGetInfo(pi_device device, pi_device_info paramName,
     std::memcpy(paramValue, &result, sizeof(cl_bool));
     return PI_SUCCESS;
   }
-
+  case PI_DEVICE_INFO_HOMOGENEOUS_ARCH: {
+    // FIXME: conservatively return false due to lack of low-level API exposing
```

Comment:
should we maybe return true for Intel GPU's already to get OpenCL backend parity with Level-Zero?

Reply:
I added a check for GPU type, but w/o a vendor check. I'm not sure how many OpenCL implementations supports device partition, but I guess it's done for homogeneous GPU only. Let me know if you want to harden the check.